src/audio/AudioListener.js

```diff
@@ -84,14 +84,19 @@ AudioListener.prototype = Object.assign( Object.create( Object3D.prototype ), {
 
 	updateMatrixWorld: ( function () {
 
-		var position = new Vector3();
```

Comment:
I'm thinking we should make an exception in this case. This function is executed many times per frame (hot path).